ts/state/ducks/types.ts

```diff
@@ -6,8 +6,12 @@ import {
 export type CallNotificationType = 'missed-call' | 'started-call' | 'answered-a-call';
 
 export type PropsForCallNotification = {
-  notificationType: CallNotificationType;
   messageId: string;
+  notificationType: CallNotificationType;
+};
+
+export type PropsForMessageRequestResponse = {
```

Comment:
This type name is duplicated in `ts/models/messageType.ts`

Reply:
removed, good catch!